src/pkg/services/m365/api/users_test.go

```diff
@@ -116,164 +113,3 @@ func (suite *UsersUnitSuite) TestEvaluateMailboxError() {
 		})
 	}
 }
-
-type UsersIntgSuite struct {
-	tester.Suite
-	its intgTesterSetup
-}
-
-func TestUsersIntgSuite(t *testing.T) {
-	suite.Run(t, &UsersIntgSuite{
-		Suite: tester.NewIntegrationSuite(
-			t,
-			[][]string{tconfig.M365AcctCredEnvs}),
-	})
-}
-
-func (suite *UsersIntgSuite) SetupSuite() {
-	suite.its = newIntegrationTesterSetup(suite.T())
-}
-
-func (suite *UsersIntgSuite) TestUsers_GetInfo_errors() {
```

Comment:
Equivalent unit tests(with mocked interfaces instead of gock) & integration tests have been added in earlier PRs for the replacement APIs -  `UserHasMailbox`, `UserHasDrive`, `UserGetMailboxInfo`.